### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by tokorv** [ReCT-Lang/ReCT-Go-Compiler#21] [General]
Hello :waffle:  :)

**Comment by Khhs167** [comment]
ho

**Reply by Khhs167** [reply]
*hi

**Comment by tokorv** [comment]
Why do you want to get symbols.TypeSymbol anyway? What you making?

**Comment by Khhs167** [comment]
You'll see. It is a big thing however

**Comment by Khhs167** [comment]
To prevent crashes

**Comment by tokorv** [comment]
Cool, I think I'm going to work on HTTP/S, WebSockets, and Json libraries in preparation for a Discord API library :+1:

**Comment by mr-cheffy** [comment]
hello!


*mark as answered pls*

**Comment by Khhs167** [comment]
Oh god this is killing me. BoundExpressionNode should have the value of the token right?

**Comment by tokorv** [comment]
What

**Reply by mr-cheffy** [reply]
What

**Comment by RedCubeDev-ByteSpace** [comment]
what

**Comment by tokorv** [comment]
Waltuh tell us what you're doing Waltuh

**Comment by mr-cheffy** [comment]


**Reply by Khhs167** [reply]
nooooo....

**Reply by mr-cheffy** [reply]
I have a screenshot ;)

**Comment by Khhs167** [comment]
I read through the issues of this repo
And... #10
